src/main/java/io/fabric8/elasticsearch/plugin/OpenShiftElasticSearchService.java

```diff
@@ -65,31 +66,41 @@ protected OpenShiftElasticSearchService(final Settings settings, final Client es
 		this.cache = cache;
 		this.aclNotifier = aclNotifer;
 		restController.registerFilter(aclFilter);
-		restController.registerFilter(new KibanaUserReindexFilter(settings, logger));
+		
+		boolean reindexEnabled = settings.getAsBoolean(OPENSHIFT_KIBANA_REWRITE_ENABLED_FLAG, OPENSHIFT_KIBANA_REWRITE_ENABLED_DEFAULT);
+		
+		if ( reindexEnabled ) {
+			restController.registerFilter(new KibanaUserReindexFilter(settings, logger));
+		}
 	}
 
 	@Override
 	protected void doStart() throws ElasticsearchException {
+		
+		boolean dynamicEnabled = settings.getAsBoolean(OPENSHIFT_DYNAMIC_ENABLED_FLAG, OPENSHIFT_DYNAMIC_ENABLED_DEFAULT);
+
```

Comment:
We additionally should provide debug log message with what the configuration was interpreted to be.  Might make it more obvious that are enabling (or not) the functionality